packages/dds/cell/README.md

```diff
@@ -2,24 +2,10 @@
 
 The `SharedCell` Distributed Data Structure (DDS) stores a single, shared value that can be edited or deleted.
 
-<!-- AUTO-GENERATED-CONTENT:START (README_INSTALLATION_SECTION:includeHeading=TRUE) -->
+This package is primarily intended as a minimal example of how distributed data structures work.
+For real-world scenarios, we recommend DDSs like [SharedTree](https://fluidframework.com./docs/data-structures/tree/) or [SharedMap](https://fluidframework.com./docs/data-structures/map/).
```

Comment:
I think the guidance is that we're not recommending any of the legacy DDS, including SharedMap, right @kashms ? So just point to SharedTree here?

Reply:
```suggestion
For real-world scenarios, we recommend using one of our other DDSs like [SharedTree](https://fluidframework.com./docs/data-structures/tree/).
```